### DISCUSSION TITLE
What would be the best command lines for minimum latency over wired LAN network?

### DISCUSSION THREAD

**Discussion by JeyP4** [roc-streaming/roc-toolkit#415] [Q&A] [answered]
Hi
Actually I tried many cmdlines e.g.
`./roc-send -vv -s rtp+rs8m:10.72.1.80:10001 -r rs8m:10.72.1.80:10002 -d pulseaudio -i echoCancel_source  --rate=16000`

`./roc-recv -vv -s rtp+rs8m:10.72.1.80:10001 -r rs8m:10.72.1.80:10002 -d pulseaudio -o alsa_output.pci-0000_00_1f.3.analog-stereo --rate=16000 --sess-latency=30ms`

But couldn't figure out best cmds for lowest possible audio latency. I am using wired LAN between 2 ubuntu 18.04 PCs.

**Comment by gavv** [comment, marked as answer]
Hi, you can take a look at this comment: https://github.com/roc-streaming/roc-toolkit/issues/255#issuecomment-531257393. I'll add it to documentation a bit later.

If you're using wired LAN, you can disable FEC to be able to reduce jitter buffer. See also this bug: #184.

Let me know if you'll need further help.

**Comment by gavv** [comment]
Closing this, feel free to re-open if you'll have more questions.
